### ISSUE TITLE
Issues with Nest 6.0

### ISSUE THREAD

**Issue by kjetilhp** [mentos1386/nest-morgan#1]
Seems to be an issues after upgrading to nest 6
```
TS2416: Property 'intercept' in type 'AbstractMorganInterceptor' is not assignable to the same property in base type 'NestInterceptor<any, any>'.
  Type '(context: ExecutionContext, call$: Observable<any>) => Observable<any>' is not assignable to type '(context: ExecutionContext, next: CallHandler<any>) => Observable<any> | Promise<Observable<any>>'.
    Types of parameters 'call$' and 'next' are incompatible.
      Type 'CallHandler<any>' is missing the following properties from type 'Observable<any>': _isScalar, source, operator, lift, and 6 more.
```

**Comment by mentos1386** [comment]
@kjetilhp Thanks for reporting this! I will update it this afternoon and release a new version.

**Comment by mentos1386** [comment]
@kjetilhp Sorry it took me so long to finish this. Released in [version 1.0.0](https://github.com/mentos1386/nest-morgan/releases/tag/v1.0.0).
